lib/read-body.js

```diff
@@ -1,14 +1,13 @@
-import { extname } from 'node:path'
-import { pipeline as _pipeline } from 'node:stream'
-import { promisify } from 'node:util'
+const { extname } = require('node:path')
+const { pipeline: _pipeline } = require('node:stream')
+const { promisify } = require('node:util')
 
-import Busboy from '@fastify/busboy'
-import { createWriteStream } from 'fs-temp'
-import hasOwnProperty from 'has-own-property'
-import _onFinished from 'on-finished'
-import FileType from 'stream-file-type'
+const Busboy = require('@fastify/busboy')
+const { createWriteStream } = require('fs-temp')
+const _onFinished = require('on-finished')
+const FileType = require('stream-file-type')
```

Comment:
(I believe that this will fail on Node.js versions that cannot `require` ESM modules)